context.go

```diff
@@ -5,6 +5,7 @@
 package dep
 
 import (
+	"fmt"
```

Comment:
Hm, not really sure what's the problem here. I did run gofmt (in fact one of the pust-push checks fails when the code is not gofmt'ed).